strings/tips.txt

```diff
@@ -175,10 +187,15 @@ As the Clown, eating bananas heals you slightly. Honk!
 As the Clown, if you lose your banana peel, you can still slip people with your PDA! Honk!
 As the Clown, if you're a Traitor and get an emag on sale (or convince another traitor), you can emag your Clown Car to unlock a variety of new functions, including the Siege Mode, which will allow you to launch your passengers, preferably directly into the Supermatter! Or into space.
 As the Clown, spice your gimmicks up! Nobody likes a one-trick pony.
+As the Clown, click with one long balloon in hand onto another to create a balloon animal! Each combination of colours has its own unique result.
 As the Clown, you can use your stamp on a sheet of cardboard as the first step of making a honkbot. Fun for the whole crew!
 As the Clown, your Grail is the mineral bananium, which can be given to the Roboticist to build you a fun and robust mech beloved by everyone.
 As the Curator, be sure to keep the shelves stocked and the library clean for crew.
+As the Curator, you can be the menace of all felenids thanks to the laser pointer you start with. Unless, which is likely, you are a weeb, in which case; try a gentler approach.
 As the Curator, you are not completely defenseless. Your whip easily disarms people, your laser pointer can blind humans and cyborgs, and you can hide items in wirecut books.
+As the Coroner, you are the medicaly certified edgelord of the station, which comes with several benefits, such as liking corpses, freely drinking medicaments meant for corpses such as formaldehyde and performing corpse-related surgeries faster.
+As the Coroner, your vendor contains appropriately edgy weaponry - you just need to hack in to get it, much like peforming an organ harvest.
```

Comment:
I would toss both of these tips out and replace them with something that just covers their general "corpse surgery" niche
```suggestion
As the Coroner, you are more comfortable working on cadavers. You can perform autopsies or harvest organs from corpses a lot faster than your Medical Doctor counterparts. Work in tandem with them by helping prepare bodies for revival.
```